spec/interpreter/native_lib/io_spec.cr

```diff
@@ -55,4 +55,31 @@ describe "NativeLib - IO Methods" do
       itr.output.to_s.should eq("")
     end
   end
+
+  describe "#gets" do
+    it "no input" do
+      itr = interpret_with_mocked_input %q(
+        IO.gets
+      ), nil
+
+      itr.stack.pop.to_s.should eq("nil")
+    end
+
+    it "a string from the IO" do
+      itr = interpret_with_mocked_input %q(
+        IO.gets               
+      ), "hello, world"
+
+      itr.stack.pop.to_s.should eq("hello, world")
```

Comment:
Same here and in the next test. Prefer `its.stack.pop.should eq(val("hello, world"))` :)